compose/ui/ui-test-junit4/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -261,6 +266,16 @@ class SkikoComposeUiTest(
         return surface.makeImageSnapshot()
     }
 
+    fun SemanticsNodeInteraction.captureToImage(): ImageBitmap {
+        val rootImage = this@SkikoComposeUiTest.captureToImage()
```

Comment:
`captureToImage` and `SemanticsNodeInteraction.captureToImage()` have different types (Image, ImageBitmap). We should keep only one. We probably should return `ImageBitmap`, so our API will depend on Compose, not Skia.

It is a breaking change, but our test API is still experimental

Reply:
I agree. Done.
Also updated some tests which used `fun captureToImage(): Image`